[internal] Simplify contributing_to_docs/tools_and_setup.adoc (#76476)
openshift/openshift-docs: contributing_to_docs/tools_and_setup.adoc

This ensures that you are tracking the remote repository to keep your local
repository in sync with it.

== Previewing the documentation locally

After you clone the documentation repository, you are ready to
install the software you need to build our content from source. All OpenShift
documentation is written in AsciiDoc, and is processed with https://github.com/redhataccess/ascii_binder[AsciiBinder],

> which is an http://asciidoctor.org/[AsciiDoctor]-based docs management system.

Author: Bad shoot, Vale.

Use AsciiBinder by running the container image.
====

=== Installing Podman on your system

Install Podman so that you can run the AsciiBinder container image on your machine.

If you are using Fedora, from a command line, enter:

[source,terminal]
----
$ sudo dnf install podman
----

If you are using another operating system, follow the link:https://podman.io/docs/installation[Podman installation docs].

=== Building the documentation

Run the container image with an AsciiBinder command to build the OpenShift documentation.

. From a command line, change to the `openshift-docs` directory.

. To build every distribution in the docs, enter the following command:
+
[source,terminal]
----
$ podman run --rm -it -v `pwd`:/docs:Z quay.io/openshift-cs/asciibinder asciibinder build
----
+

> The `asciibinder build` command runs within the image. The AsciiDoc files are transformed into HTML files in your local repository under the `_preview` directory, with a documentation set generated for every distribution.

Author: ```suggestion
The `asciibinder build` command runs within the image. The AsciiDoc files are transformed into HTML files in your local repository under the `_preview` directory. By default, AsciiBinder generates a documentation set for each distribution.
```

> TIP: To choose which distributions to build, use the `-d` option in the command. For example, to only build  `openshift-enterprise`, change `asciibinder build` to `asciibinder build -d openshift-enterprise`.

Author: ```suggestion
TIP: To choose which link:https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#product-title-and-version[distributions] to build, use the `-d` option in the command. For example, to only build  `openshift-enterprise`, change `asciibinder build` to `asciibinder build -d openshift-enterprise`.
```
